src/App.tsx

```diff
@@ -27,29 +42,44 @@ const App: React.FC = () => {
     console.log(query);
   }
 
+  const handleSortChange = (newSort: string) => {
+    setSelectedSort(newSort);
+    console.log(`Sorting movies by: ${newSort}`);
+  };
```

Comment:
**suggestion:** As we pass this function as a prop, it's better to wrap it in `useCallback`

Reply:
done